x-pack/legacy/plugins/maps/public/kibana_services.js

```diff
@@ -7,12 +7,12 @@
 import {
   getRequestInspectorStats,
   getResponseInspectorStats,
-} from '../../../../../src/legacy/ui/public/courier';
+} from '../../../../../src/legacy/core_plugins/data/public';
 import { esFilters } from '../../../../../src/plugins/data/public';
+export { SearchSource } from '../../../../../src/plugins/data/public';
```

Comment:
This is broken as is (can't resolve SearchSource). I think you want this: 

`export { SearchSource } from '../../../../../src/legacy/core_plugins/data/public/search/search_source';`

Also, small nit: would you mind moving it down to be clustered with the exports so we keep imports and exports together?

Reply:
I accidentally exported this from the wrong path, due to this PR I'm working on 
https://github.com/elastic/kibana/pull/55430